js/Dom.js

```diff
@@ -35,9 +35,8 @@ function addEvent(obj,type,fn)
 		obj["e"+type+fn] = fn;
 		obj[type+fn] = function(){obj["e"+type+fn](window.event);};
 		obj.attachEvent("on"+type,obj[type+fn]);
-	} else {
+	} else
 		obj.addEventListener(type,fn,false);
-	}
```

Comment:
uuups. nonono this is super error prone. Even if one line is part of the else branch it should have braces.